site/_docs/reference.md

```diff
@@ -2700,12 +2700,14 @@ In the following:
 | s | BIT_GET(value, position)                       | Returns the bit (0 or 1) value at the specified *position* of numeric *value*. The positions are numbered from right to left, starting at zero. The *position* argument cannot be negative
 | b | CEIL(value)                                    | Similar to standard `CEIL(value)` except if *value* is an integer type, the return type is a double
 | m s | CHAR(integer)                                | Returns the character whose ASCII code is *integer* % 256, or null if *integer* &lt; 0
+| s | CHR(integer)                                   | Returns the character whose ASCII code is *integer* % 256, or null if *integer* &lt; 0
```

Comment:
I know spark is different from other engines.
But I don't think we should add same function in `reference.md`

Reply:
I am not sure, have any suggestions for this case?

Reply:
@JiajunBernoulli @YiwenWu If some functions of spark are different from other sql engines, I do not recommend putting them in this jira, as this can easily cause confusion. We need to open a new jira to discuss it, what is the difference and what negative tests need to be added?

Reply:
@caicancai  The difference：
- Spark [CHR](https://spark.apache.org/docs/latest/api/sql/index.html#chr)： Returns the ASCII character;
- Currently defined CHR: returns the character UTF-8 code